lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -485,17 +513,20 @@ WasmBytecodeGenerator::EmitBlock()
     if (m_reader->IsBinaryReader())
     {
         UINT blockCount = m_reader->m_currentNode.block.count;
-        if (blockCount <= 0)
+        if (blockCount > 0)
         {
-            throw WasmCompilationException(_u("Invalid block node count"));
```

Comment:
We'll have to verify if an empty block is valid or not.
I changed it because I came across a test case that uses empty block. The test might have been out of date though.